README.md

```diff
@@ -397,7 +384,7 @@ When TLS connection is configured, `ca_cert_src` and/or `ca_cert` is required.
 To configure the certificate of the logging system, `cert_src` and/or `cert` is required.
 To configure the private key of the logging system, `private_key_src` and/or `private_key` is required.
 
-```
+```yaml
```

Comment:
This needs to be reformatted entirely - see https://github.com/linux-system-roles/logging/pull/345